python/pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools==65.6.3", "conan==1.62", "cmake==3.26.4", "setuptools_scm[toml]>=6.2"]
+requires = ["setuptools==65.6.3", "conan==1.62", "cmake==3.26.4", "setuptools_scm[toml]>=8.0.0"]
```

Comment:
We should either remove support for the Python 3.7 or stick to the `setuptools_scm` version `7.1.0`.

Reply:
Yes, I've already changed this line to 7.1.0. My bad.